### PR TITLE
BL-293-as-a-case-manager-i-want-to-export-the-case-notes-so-that-we-can-use-them-to-submit-to-foundational-community-support-fcs

### PR DESCRIPTION
**Pull request by JordonP123**
The goal was to retrieve all the notes given for a specific client.  I added a button on the case-details page that routes to a new component.  That component is a more formal document (which is why it is not dark mode).  The component looks correct in print/pdf format.  I hit the api in the api folder, which updates based on the id given.  The ID is not updatable yet, as we need redux or props passed down from the parent to know which client we are viewing.  I passed in dummy data for now.

### CODE REVIEW

**Comment by JordonP123** [comment]
https://www.loom.com/share/a47705ceae5643e4ae8cb47d7c22414f

File: src/components/common/CaseDetails/printNotes.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from 'react';
```
Comment (Kobramall) [inline src/components/common/CaseDetails/printNotes.js:1]: I don't see any errors for what I can tell

File: src/styles/css/styles.css [hunk 2]
```diff
@@ -852,6 +852,54 @@ td.ant-table-column-sort {
   background-color: #101010;
 }
 
+.printNotesHeader{
```
Comment (Kobramall) [inline src/styles/css/styles.css:855]: Nothing wrong with styling either

**Review by Kobramall — APPROVED** [review]

**Review by Valeriusdev — APPROVED** [review]
Good job, Jordan! Also you explained well what you did in your Loom video.

File: src/api/index.js [hunk 3]
```diff
@@ -49,4 +49,15 @@ const getProfileData = authState => {
   }
 };
 
-export { sleep, getExampleData, getProfileData, getDSData };
+const getClientsNotes = clientId => {
```
[the remark this answers is not in the document]
Comment (JordonP123) [reply src/api/index.js:52]: Yes, first tested the endpoints using postman, then ran the data in through an Axios call.

**Pull request merged**